### PR TITLE
✨ [amp-google-read-aloud-player] Adds base src

### PR DESCRIPTION
**Pull request by mhalabi-google**
See [I2I #31013](https://github.com/ampproject/amphtml/issues/31013).
cc: @alanorozco

### CODE REVIEW

File: extensions/amp-google-read-aloud-player/0.1/test/test-amp-google-read-aloud-player.js [hunk 1]
```diff
@@ -137,6 +138,24 @@ describes.realWin(
         );
       });
 
+      it('sets optinal iframe base src', async () => {
```
Comment (alanorozco) [inline extensions/amp-google-read-aloud-player/0.1/test/test-amp-google-read-aloud-player.js:141]: nit: 
```suggestion
      it('sets optional iframe base src', async () => {
```

**Review by alanorozco — COMMENTED** [review]
Could you please undo the changes that are exactly like #37666? Happy to merge afterwards. Thank you!

**Review by mhalabi-google — COMMENTED** [review]
@alanorozco Thank you for the quick review!

QQ: The base URL is not really "data", it is not a param that is sent to the player.

What do you say about renaming it to just `base-src` (similar to `src` param in `amp-video-iframe`)?

**Comment by alanorozco** [comment]
@mhalabi-google Yeah, I support using something other than `data-`. It's cleaner since you wouldn't have to remove the object key. I also mentioned offline that you shouldn't allow the attribute in validation, but there's no good way to forbid a `data-*` attribute. 

I would name it `dev-src` or anything that specifies that it's a dev-only feature. Make sure that you don't list it in validation, it would be forbidden by default.

**Comment by mhalabi-google** [comment]
> @mhalabi-google Yeah, I support using something other than `data-`. It's cleaner since you wouldn't have to remove the object key. I also mentioned offline that you shouldn't allow the attribute in validation, but there's no good way to forbid a `data-*` attribute.
> 
> I would name it `dev-src` or anything that specifies that it's a dev-only feature. Make sure that you don't list it in validation, it would be forbidden by default.

We may actually want to expose it in the future for clients to be, for example if we want to enable them to test a beta version of the player or to set the version of the player they want to embed in their site.

Therefore I suggest using `base-src`.

WDYS?

**Comment by alanorozco** [comment]
@mhalabi-google I see. Sounds like the semantics are exactly like `amp-video-iframe`'s `src`, so just for consistency I'd suggest `src`.

File: extensions/amp-google-read-aloud-player/0.1/amp-google-read-aloud-player.js [hunk 2]
```diff
@@ -89,17 +92,22 @@ export class AmpGoogleReadAloudPlayer extends AMP.BaseElement {
...
   /** @return {string} */
   getIframeSrc_() {
-    const src = addParamsToUrl(
-      IFRAME_BASE_URL,
-      getDataParamsFromAttributes(
-        this.element,
-        /* opt_computeParamNameFunc = */ undefined,
-        '^(.+)'
-      )
+    const dataParams = getDataParamsFromAttributes(
```
Comment (alanorozco) [inline extensions/amp-google-read-aloud-player/0.1/amp-google-read-aloud-player.js:103]: nit: Just for diff's sake I would revert this method fully.

Comment (mhalabi-google) [reply extensions/amp-google-read-aloud-player/0.1/amp-google-read-aloud-player.js]: Done.

**Review by alanorozco — APPROVED** [review]
Thank you!

**Comment by mhalabi-google** [comment]
Thank you @alanorozco for the review!

**Pull request merged**